Dockerfile

```diff
@@ -1,9 +1,9 @@
-FROM openjdk:alpine
+FROM openjdk:12
```

Comment:
(Not blocking/Out of interest) why no longer alpine?

And why using the JDK when only using JRE stuff (see https://adoptopenjdk.net/about.html - e.g. `adoptopenjdk/openjdk12:jre-12.33`

Reply:
The alpine image didn't work on 12, gave me an error saying the software is too new, so I guess it's still outdated for some reason. Could try a Java 11 JRE image, do you recommend/have one?

Reply:
I only found the one above :P `adoptopenjdk/openjdk12:jre-12.33`.

But that is indeed a good reason :D